src/registry/routes/publish.ts

```diff
@@ -65,20 +65,20 @@ export default function publish(repository: Repository) {
         res.status(200).json({ ok: true });
       } catch (err: any) {
         if (err.code === 'not_allowed') {
-          res.errorDetails = `Publish not allowed: ${err.msg}`;
-          res.status(403).json({ error: err.msg });
+          res.errorDetails = `Publish not allowed: ${err}`;
```

Comment:
I think the proposed solutions included more lines to transform the errormessage

Reply:
Hey Ricardo,

I should have mentioned it while raising the PR.
Anyways in the proposed solution
I was checking the `error object` if it has `msg` field or `message` field, if none then I was `JSON.stringify`ing the object and setting it everywhere

But it made sense to me, to just send the whole error object.

Let me know if you think otherwise or why should we not do this.

Reply:
Well I'm thinking that we probably shouldn't return a stack trace on a production environment API

Reply:
Makes sense.
I have made the changes that match the same as I had proposed in the issue.

Reply:
Do you think we should also include logging errors ?

Reply:
Do you think, will it be better to use `res.conf.debug` instead of local?

Reply:
There is no conf.debug property. Local should be fine.

Reply:
Cool, I have made the changes